README.project.md.erb

```diff
@@ -12,50 +12,74 @@ git clone <%= config.git_url %>
 
 ## Development Environment
 
-### Local development environment
+### Getting Started
 
-#### Docker setup
-If you are on OSX, install [Docker for Mac](https://docs.docker.com/docker-for-mac/install/).
+#### Prerequisites
 
-If you are on Linux, use your package manager to install `docker`.
+##### General
 
-If you are on Windows, install [Docker for Windows](https://docs.docker.com/docker-for-windows/install/).
+- A GitHub Token
+  - This can be generated at [GitHub](https://github.com/settings/tokens).
+- Access to LastPass
+- Access to AWS
+  - If you do not have access you can request it from support.
+- Tideways API Key
+  - If you wish to run tideways profiling on your development environment, you will find this in LastPass.
 
-You should also install dockrsync from [dockrsync](https://github.com/webtatic/dockrsync).
+##### Docker
 
-#### Project setup
-1. Run `dockrsync setup` and fill in the questions:
-   - The default service should be set as "web".
-   - The anybar question can be skipped over by hitting enter.
-   - We recommend choosing "yes" for the delete flag question.
-2. Copy `docker.env.dist` to `docker.env`
-3. Copy `docker-compose.override.yml.dist` to `docker-compose.override.yml`
-4. Fill in docker.env's secrets - do not commit this file!
+- A working docker setup
+  - On MacOS, use [Docker for Mac](https://docs.docker.com/docker-for-mac/install/)
+  - On Linux, add the official Docker repository and install the "docker-ce" package.
+
+#### Setup
+
+1. Copy `docker.env.dist` to `docker.env`
+2. Fill in docker.env's secrets - do not commit this file!
    - TIDEWAYS_API_KEY can be found in LastPass
    - AWS_ACCESS_KEY_ID is found in your Hem configuration. Check `~/.hem/config.yaml` and `~/.hobo/config.yaml`.
    - AWS_SECRET_ACCESS_KEY should be found wherever AWS_ACCESS_KEY_ID was.
-5. Run the following:
+3. Run the following:
    - for bash/zsh: `eval "$(echo $(printf "%s " $(cat docker.env)) docker-compose build)"`
    - for fish: `eval env (echo (printf "'%s' " (cat docker.env)) docker-compose build)`
-6. Run `docker-compose up -d`
-7. Wait for the build to finish, you can follow along with `docker-compose logs -f web`
-8. Run `docker-compose exec web bash` to log in to your local "web" container.
-9. Run this to let you see the site from a nice domain:
+4. Run this to let you see the site from a nice domain:
     - On Linux/OSX: `echo "127.0.0.1 <%= config.hostname %>" | sudo tee -a /etc/hosts`
     - On Windows, open up `C:\Windows\system32\etc\HOSTS` in your favourite text editor as admin and add `127.0.0.1 <%= config.hostname %>`
-10. Run `dockrsync watch` to watch for changes to your local filesystem and upload them to the web container.
-11. The site will be available at [https://<%= config.hostname %>](https://<%= config.hostname %>). Happy coding!
+5. Setup docker-compose.override config and start docker-compose up:
+
+```bash
+bundle install
+
+rake docker:setup
```

Comment:
This gives:
```
$ rake docker:setup
rake aborted!
Don't know how to build task 'docker:start' (see --tasks)
/Users/kevans/.hem/gems/ruby/2.2.0/bundler/gems/inviqa-rake-tasks-docker-sync-e1dae4989882/lib/rake-tasks-docker-sync/rake_task.rb:59:in `<top (required)>'
/Users/kevans/Repositories/DEVOPS-564-docker-setup-and-sync/Rakefile:3:in `require'
/Users/kevans/Repositories/DEVOPS-564-docker-setup-and-sync/Rakefile:3:in `<top (required)>'
/Users/kevans/.hem/gems/ruby/2.2.0/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
/usr/local/bin/ruby_executable_hooks:15:in `eval'
/usr/local/bin/ruby_executable_hooks:15:in `<main>'
(See full trace by running task with --trace)
```

Reply:
The following PR's fix this
https://github.com/inviqa/inviqa-rake-tasks-docker/pull/5
https://github.com/inviqa/inviqa-rake-tasks-docker-sync/pull/1

Reply:
Approved both :+1:, will give another go after they are merged and Gemfile.lock updated

Reply:
done @kierenevans

Reply:
This is now working with delegated mode and sync mode with one hiccup.

As I had `RAKE_USE_DOCKER_SYNC=true` defined in my shell, `docker-compose up -d` from the README didn't work due to:
```
➜  DEVOPS-564-docker-setup-and-sync-2 git:(develop) docker-compose up -d
Creating network "devops564dockersetupandsync2_default" with the default driver
ERROR: Volume DEVOPS-564-docker-setup-and-sync-2-sync declared as external, but could not be found. Please create the volume manually using `docker volume create --name=DEVOPS-564-docker-setup-and-sync-2-sync` and try again.
```

I used the second version, `rake docker:up` and it worked. Do we need to port the RAKE_USE_DOCKER_SYNC documentation from #90 and advocate the `rake docker:up` command by default?

Reply:
RAKE_USE_DOCKER_SYNC docs were on #101 instead of 90.